lib/roster.ts

```diff
@@ -0,0 +1,46 @@
+import { AscensionLevel, CharacterClass, Faction } from "./characters";
+
+export enum Category {
+  Starter = "starter",
+  Seasonal = "seasonal",
+}
+
+export type EquipmentClass = {
+  name: string;
+  imageUrl: string;
+};
+
+export type Season = {
+  key: string;
+  name: string;
+};
+
+export type Level = {
+  key: string;
+  seasonName: string;
+  maxLevel: number;
+  active: boolean;
+  level: number;
+};
+
+export type Artifact = {
+  key: string;
+  imageUrl: string;
+  label: string;
+  category: Category;
```

Comment:
this should probably just be a `Season`

Reply:
So the `Season` has more data here than we need.. it's more of a flag whether it's seasonal or starter. I could just make it a boolean instead?

Reply:
`Category` is fine, especially if we aren't going to be handling multiple season rosters like you pointed out above